tests/unit/test_api.py

```diff
@@ -0,0 +1,33 @@
+
+from unittest import TestCase
```

Comment:
@jbzdak there's a module for api_tests: https://github.com/open-craft/xblock-group-project-v2/blob/master/tests/unit/project_api/test_project_api.py